Documentation/data-format.md

```diff
@@ -0,0 +1,69 @@
+# Adobe Experience Platform Edge Bridge for iOS
+
+## Data Format
+
+The AEP Edge Bridge extension maps the data sent in `trackAction` and `trackState` API calls to a specific format known to Analytics which requires no additional server-side mapping. This document outlines the data format applied to the tracking data.
+
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" element of the Experience Event sent to the Edge Network.
+
+
+| Data | Key in dispatched request | Comments |
+| --- | --- | ---
+| action | `data.__adobe.analytics.linkName` | Additionally, the field `data.__adobe.analytics.linkType` with value `lnk_o` is automatically included. |
+| state | `data.__adobe.analytics.pageName` | |
+| contextdata | `data.__adobe.analytics.contextData` | Context data is a map which includes the custom keys and values specified in the `trackAction` and `trackState` API calls. |
+| prefixed contextdata | `data.__adobe.analytics` | Context data keys prefixed with `&&` must be known to Analytics and are case sensitive. When mapped to the event, the key's name does not include the "&&" prefix.|
+| app identifier | `data.__adobe.analytics.contextData.a.AppID` | The application identifier is automatically added to every tracking event. Note the key name is "a.AppID".|
+| customer perspective | `data.__adobe.analytics.cp` | The customer perspective is automatically added to every tracking event. The values are either "foreground" or "background". |
+
+### Examples
+
+Given the track action call: 
+
+```swift
+MobileCore.track(action: "action name", data: ["key": "value", "&&products": ";Running Shoes;1;69.95;event1|event2=55.99;eVar1=12345"])
+```
+The resulting Experience Event has the following `data` element:
```

Comment:
data element may e confusing for Data Collection users. How about: The resulting Experience Event has the following payload: